klayout/drc/rule_decks/mim_b.drc

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if MIM_OPTION == 'B'
+  #================================================
+  #-------------MIM CAPACITOR OPTION B-------------
+  #================================================
+
+  logger.info('Starting MIM Capacitor Option B derivations')
+
+  topmin1_metal_fusetop = topmin1_metal.interacting(fusetop)
+  mimtm_virtual = fusetop.sized(1.06.um).and(topmin1_metal_fusetop)
+
+  # Rule MIMTM.1: Minimum MiM bottom plate spacing to the bottom plate metal
+  ## (whether adjacent MiM or routing metal) is 1.2µm.
+  logger.info('Executing rule MIMTM.1')
+  mimtm1_l1 = topmin1_metal.separation(mimtm_virtual, transparent, 1.2.um)
+  mimtm1_l1.output('MIMTM.1', "MIMTM.1 : Minimum MiM bottom plate spacing to the bottom plate metal
+                    (whether adjacent MiM or routing metal): 1.2µm")
+  mimtm1_l1.forget
+
+  # Rule MIMTM.2: Minimum MiM bottom plate overlap of Vian-1 layer.
+  ## [This is applicable for Vian-1 within 1.06um oversize of FuseTop layer
+  ## (referenced to virtual bottom plate)] is 0.4µm.
+  logger.info('Executing rule MIMTM.2')
+  mimtm_via = top_via.overlapping(mimtm_virtual)
+  mimtm2_l1 = mimtm_via.enclosed(topmin1_metal, 0.4.um, euclidian).polygons(0.001.um)
```

Comment:
> polygons(0.001.um)

is it used to convert to edge to polygons, should we add a comment?

Reply:
@proppy   yes,  `.polygons` command is used to convert edges to polygons to be able to ORing it with polygons for the rest of the rule.

I don't think we need to explain that, it's a common command in Klayout.